source/documentation/services/airflow/index.md

```diff
@@ -70,37 +76,51 @@ After your request is granted, you will be added to a GitHub team that will give
 
 1. Create a repository using one of the provided runtime templates:
 
-    > You can create this repository in either the [`ministryofjustice`](https://github.com/ministryofjustice/) or [`moj-analytical-services`](https://github.com/moj-analytical-services/) GitHub organisation
-    >
-    > Repository standards, such as branch protection, are out of scope for this guidance
-    >
-    > For more information on runtime templates, please refer to [runtime templates](#runtime-templates)
+- [Python Airflow Template](https://github.com/new?template_name=analytical-platform-airflow-python-template&template_owner=ministryofjustice)
+
+- [R Airflow Template](https://github.com/new?template_name=analytical-platform-airflow-r-template&template_owner=ministryofjustice)
+     
+  > You can create this repository in either the [`ministryofjustice`](https://github.com/ministryofjustice/) or [`moj-analytical-services`](https://github.com/moj-analytical-services/) GitHub organisation
+  >
+  > Repository standards, such as branch protection, are out of scope for this guidance
+  >
+  > For more information on runtime templates, please refer to [runtime templates](#runtime-templates)
+
+2\. Add your code to the repository, including the script(s) your want Airflow to run and a file for your package management
 
-    [Python](https://github.com/new?template_name=analytical-platform-airflow-python-template&template_owner=ministryofjustice)
+3\. Update the `Dockerfile` instructions to copy your code into the image, install packages required to run, and call the script(s) to run. For example, for Python:
```

Comment:
Thought it might be a good example to add it here rather than get people to go to a separate link to find an example?